libs/auth/src/user.ts

```diff
@@ -0,0 +1,75 @@
+import { prisma } from '@nmit-coursition/db'
+import type { cas__user } from '@prisma/client'
```

Comment:
Interesting, it's the first time I see such import from Prisma.

Reply:
It is mine strict-defined response